gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/DiagnosticsManifestPlugin.java

```diff
@@ -117,7 +117,7 @@ private static void configureProjectDependencyTransform(Project project) {
         project.getDependencies().registerTransform(SelectSingleFile.class, details -> {
             details.getParameters().getPathToExtract().set(FILE_TO_EXTRACT);
 
-            details.getFrom().attribute(artifactType, ArtifactTypeDefinition.JVM_RESOURCES_DIRECTORY);
```

Comment:
Will this removal impact folks who check in manually-generated resource manifests?

Reply:
We may want to set up two transforms, one for each

Reply:
Yeah I'm trying to figure this out.

I had tried setting both here - poking around the Gradle code it looked like they would be combined. But that didn't appear to work.

https://github.com/gradle/gradle/blob/5d5de431f8d0c53f2c3e732753a066362eac2b20/subprojects/core/src/main/java/org/gradle/api/internal/attributes/DefaultMutableAttributeContainer.java#L58-L63

https://github.com/gradle/gradle/blob/5d5de431f8d0c53f2c3e732753a066362eac2b20/subprojects/core/src/main/java/org/gradle/api/internal/attributes/DefaultImmutableAttributesFactory.java#L98-L110

Might have to poke at this more.

Reply:
Yeah it's last write wins. So if you have

```
details.getFrom().attribute(artifactType, ArtifactTypeDefinition.JVM_CLASS_DIRECTORY);
details.getFrom().attribute(artifactType, ArtifactTypeDefinition.JVM_RESOURCES_DIRECTORY);
```

then it only picks up manfiests from the resources directory.

Maybe we just need to register a separate transform.